skyportal/tests/api/test_newsfeed.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from skyportal.tests import api
+
+
+def test_token_user_retrieving_source_without_nested(
+    view_only_token, public_group, upload_data_token
+):
+    obj_id = str(uuid.uuid4())
+    status, data = api(
+        "POST",
+        "sources",
+        data={
+            "id": obj_id,
+            "ra": 235.22,
+            "dec": -23.33,
+            "redshift": 3,
+            "group_ids": [public_group.id],
+        },
+        token=upload_data_token,
+    )
+    assert status == 200
+
+    params = {'numItems': 100}
```

Comment:
What happens if there are more than 100 sources in the test DB because of previous tests?

Reply:
Then this would fail. I can bump this to 1000.